packages/core/lib/src/widgets/ui/navigation/app_drawer.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:core/core.dart';
+import 'package:flutter/material.dart';
+
+/// App drawer widget
+class AppDrawer extends StatelessWidget {
+  /// Creates App drawer widget
+  const AppDrawer({
+    super.key,
+    required this.user,
+  });
+
+  /// The logged in user
+  final User user;
+
+  /// List of drawer menu items with localized labels
+  List<MenuListItemData> getDrawerMenuItems(BuildContext context) {
+    return [
+      MenuListItemData(
+        label: AppLocalizations.of(context)!.profile,
+        icon: TwitterIcons.user,
+      ),
+      MenuListItemData(
+        label: AppLocalizations.of(context)!.lists,
+        icon: TwitterIcons.list,
+      ),
+      MenuListItemData(
+        label: AppLocalizations.of(context)!.topics,
+        icon: TwitterIcons.topics,
+      ),
+      MenuListItemData(
+        label: AppLocalizations.of(context)!.notifications,
+        icon: TwitterIcons.bell,
+      ),
+      MenuListItemData(
+        label: AppLocalizations.of(context)!.bookmarks,
+        icon: TwitterIcons.bookmark,
+      ),
+    ];
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final listItems = getDrawerMenuItems(context);
+
+    return Container(
+      width: 320,
+      decoration: BoxDecoration(
+        border: BorderDirectional(
+          end: BorderSide(color: Theme.of(context).dividerColor),
+        ),
+      ),
+      child: SafeArea(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 20, vertical: 20),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Avatar(imageUrl: user.avatarUrl),
+                  const SizedBox(height: 10),
+                  DisplayName(user: user),
+                  Username(user: user),
+                  const SizedBox(height: 20),
+                  Row(
+                    children: [
+                      Text(
+                        '${user.following}',
+                        style: Theme.of(context).textTheme.bodyText1,
+                      ),
+                      const SizedBox(width: 5),
```

Comment:
BTW, I think we can have a Widget e.g. `TwitterGap.small()` for `SizedBox(width: 5)` and `TwitterGap.medium()` for `SizedBox(width: 20)`. Therefore we can make sure to meet the Design Grid consistently. What do you think?

Reply:
Agreed 👍 Added an issue.